README.md

```diff
@@ -1,15 +1,11 @@
 ## TSD API Client
 
-Python client for [TSD HTTP API](https://test.api.tsd.usit.no/v1/docs/tsd-api-integration.html).
+Python client for TSD API.
 
 ## Install
 
-Download the [latest release](https://github.com/unioslo/tsd-api-client/releases), unzip the archive, and then install as follows:
-
 ```bash
-cd tsd-api-client
-pip3 install -r requirements.txt
-python3 setup.py install
+pip3 install https://github.com/unioslo/tsd-api-client/archive/v2.4.0.zip
```

Comment:
I like the simplification but does GitHub really not provide a way to link the latest tag's tarball/zip? That's inconvenient.

Reply:
will check it out

Reply:
Looks like they do not :|

Reply:
Will just have to live with updating the readme with new releases then. Guess we should eventually publish this on pypi.